core/src/main/java/com/googlecode/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -99,9 +99,8 @@ public void setWrapper(Wrapper wrapper) {
                   tomcatContainer = (TomcatContainer) obj;
                   tomcatContainer.setWrapper(wrapper);
                   break;
-                } else {
-                  logger.debug("Cannot bind " + className + " to " + serverInfo);
                 }
+                logger.debug("Cannot bind " + className + " to " + serverInfo);
```

Comment:
Well, this is only triggering when the `if` block has `return` or `continue` or `break` or any other flow control statement. In those cases, whether what follows is in an `else` block or not really doesn't matter. The flow will be identical. That's all I mean. You could argue that either way is fragile.

Eclipse is very opinionated. I wouldn't get too caught up trying to fix everything the warn you about.